src/ui/EditorTabManager.cpp

```diff
@@ -87,9 +87,12 @@ EditorTabManager::GetToolTipText(int32 index)
 	BString label("");
 	Editor* editor = EditorAt(index);
 	if (editor) {
-		label << editor->Name();
```

Comment:
In case of a long name and many opened files, having the name in the tooltip could be useful. See picure:
![2024-02-09_14-28](https://github.com/Genio-The-Haiku-IDE/Genio/assets/1453326/c44c4599-e54d-4b7f-af39-722848b13fe9)

We anyway  add the FilePath  if you think it's necessary

Reply:
And of course there had to be an obvious reason that I couldn't see by myself :flushed: 

I didn't completely remove the file name, it's still in the path, although it's more difficult to get it if that's all you want: you have to scan to the end of the line instead of it jumping at your eyes.
![path](https://github.com/Genio-The-Haiku-IDE/Genio/assets/13631068/b6e64542-df56-484f-9387-81ba918f29af)

But *necessary* is a quite a strong word; it would at most be a little helper with other real solutions needed, so I'm OK with dropping this patch.

I got the idea when I was browsing code in a project I was not familiar with and after a few *go to definition* jumps I wanted to know where I was to open related files. I guess the real solution here would be an action in the context menu to expand the item and it scroll into view in the project browser. And that's just me, anyway.

Another case would be having several files with the same name in a project, for example arch-dependent code managed in per-arch folders. Still, if you just want to know which of the files is open, we are back to the previous case. And if you want to distinguish among open files when you are editing several of them, the real solution would be for the titles in the tabs to show the difference, not to hover and wait for a tooltip.

Reply:
I thought the change was displaying only the path (without the filename). I'm ok with your change would be nice to have a configuration flag to have both version (with and without fullpath) but let's start to merge this annd let's see.

Fully agree on adding a "Locate in browser" menu option to select the file in the project browser.